extensions/azurecore/src/extension.ts

```diff
@@ -91,6 +91,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 	pushDisposable(vscode.workspace.onDidChangeConfiguration(e => onDidChangeConfiguration(e), this));
 	registerAzureResourceCommands(appContext, [azureResourceTree, connectionDialogTree]);
 	azdata.dataprotocol.registerDataGridProvider(new AzureDataGridProvider(appContext));
+	vscode.commands.registerCommand('azure.dataGrid.openInAzurePortal', async (item: azdata.DataGridItem) => {
+		await vscode.env.openExternal(vscode.Uri.parse('https://microsoft.com'));
```

Comment:
Is this is temporary implementation? If so then suggest adding a TODO comment

Reply:
Yes it's temporary. I personally don't find TODOs useful and will be replacing this as my next task so didn't bother putting anything there.